docs/source/api_reference/molecular_models.csv

```diff
@@ -11,7 +11,7 @@ MEGNetModel,`ref <https://arxiv.org/abs/1812.05055>`_,Classifier/ Regressor,,PyT
 MPNNModel,`ref <https://arxiv.org/abs/1704.01212>`_,Classifier/ Regressor,MolGraphConvFeaturizer,DGL/PyTorch,
 PagtnModel,`ref <https://arxiv.org/abs/1905.12712>`_,Classifier/ Regressor,PagtnMolGraphFeaturizer MolGraphConvFeaturizer,DGL/PyTorch,
 Smiles2Vec,`ref <https://arxiv.org/abs/1712.02034>`_,Classifier/ Regressor,SmilesToSeq,Keras,
-TextCNNModel,`ref <https://arxiv.org/abs/1705.10843>`_,Classifier/ Regressor,,Keras,
+TextCNNModel,`ref <https://arxiv.org/abs/1705.10843>`_,Classifier/ Regressor,,PyTorch,
```

Comment:
Can you convert this to `Keras/PyTorch` as both implementations exist in the repo

Reply:
Done.